demo-app/src/main/java/cash/z/ecc/android/sdk/demoapp/SharedViewModel.kt

```diff
@@ -1,21 +1,38 @@
 package cash.z.ecc.android.sdk.demoapp
 
-import androidx.lifecycle.ViewModel
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
 import cash.z.ecc.android.bip39.Mnemonics
+import cash.z.ecc.android.sdk.demoapp.util.fromResources
 import cash.z.ecc.android.sdk.internal.twig
+import cash.z.ecc.android.sdk.model.BlockHeight
+import cash.z.ecc.android.sdk.model.ZcashNetwork
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
+import kotlinx.coroutines.runBlocking
 
 /**
  * Shared mutable state for the demo
  */
-class SharedViewModel : ViewModel() {
+class SharedViewModel(application: Application) : AndroidViewModel(application) {
 
     private val _seedPhrase = MutableStateFlow(DemoConstants.INITIAL_SEED_WORDS)
 
+    private val _blockHeight = MutableStateFlow<BlockHeight?>(
```

Comment:
This helps make the demo app much much faster.  It means that old transactions aren't found in the wallet history, but I think that's OK because otherwise we're waiting _forever_ just to do some simple testing.  We can revert this later when faster syncing is implemented.